src/index.ts

```diff
@@ -89,6 +104,8 @@ function initializeGraph() {
 }
 
 initializeNavbar()
+initializeLegend()
+LegendHTML.initialize()
```

Comment:
Put this function call inside `initializeLegend()`.

Reply:
I moved LegendHTML.initialize() within the initializeLegend() function.